DevCycleTests/Models/DVCClientTest.swift

```diff
@@ -139,9 +139,52 @@ class DVCClientTest: XCTestCase {
         wait(for: [expectation], timeout: 1.0)
     }
     
+    func testDisableCustomEventLogging() {
+        let expectation = XCTestExpectation(description: "test disableCustomEventLogging")
+        let options = DVCOptions.builder().disableAutomaticEventLogging(false).flushEventsIntervalMs(10000).build()
```

Comment:
should this be testing that setting the option to true actually disables the events? or are you just testing that the option exists?

Reply:
this was a mistake, fixed it